.NET/Microsoft.Recognizers.Definitions/English/DateTimeDefinitions.cs

```diff
@@ -50,7 +50,7 @@ public static class DateTimeDefinitions
 		public const string WeekOfRegex = @"(week)(\s*)(of)";
 		public const string MonthOfRegex = @"(month)(\s*)(of)";
 		public const string MonthRegex = @"(?<month>April|Apr|August|Aug|December|Dec|February|Feb|January|Jan|July|Jul|June|Jun|March|Mar|May|November|Nov|October|Oct|September|Sept|Sep)";
-		public const string DateYearRegex = @"(?<year>19\d{2}|20\d{2}|((9\d|0\d|1\d|2\d)(?!\s*\:)))";
+		public static readonly string DateYearRegex = $@"(?<year>19\d{{2}}|20\d{{2}}|((9\d|0\d|1\d|2\d)(?!(\s*((\:)|{AmDescRegex}|{PmDescRegex})))))";

```

Comment:
Why are AmDesc and PmDesc references in a regex called DateYear?

Reply:
This is a looking behind assertion. In some cases we will extract two digits as year like 11/25/16, the 16 here equals to 2016. Using the assertion here to determine if is not connected with the am or pm.

Reply:
I see. Can you add this as a comment in the line immediately before this regex in the YAML file?
Comments in YAML start with #.
I think this is very relevant for people translating it to other languages to know.